### PR TITLE
partials/tabs/home.html: use href instead of tc.setTab

### PR DESCRIPTION
**Pull request by Arsfiqball**
Change underline tab indicator position
when user click 'Check the available bears!'

Closes https://github.com/coala/landing-frontend/issues/164

### CODE REVIEW

**Comment by Arsfiqball** [comment]
@IpshitaC updated :)

**Comment by IpshitaC** [comment]
ack d90770d1e7e1de58eac4ed139bc92f55cd834890

File: partials/tabs/home.html [hunk 1]
```diff
@@ -115,7 +115,7 @@ <h2 class="fine">get started</div>
                     <div btf-markdown ng-include="'data/snippets/' + sc.snip[sc.lang_selected]" class="card card-content"></div>
                     <p>
                         Need more powerful analysis?
-                        <a ng-click="tc.setTab('/languages')">
+                        <a onclick="$('#tab-languages').trigger('click')">
```
Comment (hemangsk) [inline partials/tabs/home.html]: Let's go with the href='https://coala.io/languages/' ?

Comment (Arsfiqball) [inline partials/tabs/home.html]: oh you make that url redirect to /#!languages?
ok, I understand now, so how about the ``id``?
``id`` does not make sense if I change it.

Comment (hemangsk) [inline partials/tabs/home.html]: Yeah ids not required if we go with href.

**Comment by Arsfiqball** [comment]
thanks @hemangsk :)

**Comment by hemangsk** [comment]
ack 2a614f4

**Comment by hemangsk** [comment]
@rultor merge

**Comment by gkrishnan724** [comment]
@hemangsk @Arsfiqball This change, reloads the whole page. Instead of routing it to languages. I don't think this is a proper fix.

**Comment by hemangsk** [comment]
@gkrishnan724, actually we tried a lot of modifications like reinitialising select tab whenever that link is clicked, turns out it was just being an overkill for just a page redirect. Do you feel you can take out some time out and check out this weird tab behaviour, where the indicators just won't work on tab change within the site.

**Comment by IpshitaC** [comment]
IMO, hardcoding absolute URLs is not a good idea as even the localhost is redirecting to coala.io/languages

**Pull request merged**